solarkraft/src/instrument.ts

```diff
@@ -18,21 +64,40 @@ export function instrumentMonitor(
 ): any {
     // Only instrument state variables that are delcared in the monitor spec
     // (otherwise we provoke type errors in Apalache Snowcat, via undeclared variables)
+    // Since `fields` and `oldFields` can have different entries, due to data storage lifetimes,
+    // we initialize all variables missing from either fields collection with Gen(1)
```

Comment:
I think the first 2 lines are obsolete.
Also changed the last sentence a bit, the clarify what it means to be "missing".

```suggestion
    // Since `fields` and `oldFields` can have different entries, due to data storage lifetimes,
    // we initialize all variables that appear in the monitor specification but are 
    // missing from either fields collection, with Gen(1).
```